python/tvm/relay/frontend/tensorflow.py

```diff
@@ -3851,11 +3851,11 @@ def _convert_operator(
     @staticmethod
     def _set_span(sym, node_name):
         span = tvm.relay.Span(tvm.relay.SourceName(node_name), 0, 0, 0, 0)
-        if isinstance(sym, _expr.Call):
+        if isinstance(sym, _expr.Call) and sym.span is None:
```

Comment:
could you add a test case for these two lines?

Reply:
I'd love to, but I don't know how to create unitest for pb parsering flow, any suggestion?  thanks

Reply:
Does this require pb parsing or directly creating a tf program would be sufficient?

Reply:
> Does this require pb parsing or directly creating a tf program would be sufficient?

You are right，where can I find an exmple? I know the basic idea is creatig a tf graph, using from_tensorflow to handle it, then check the output graph ir.

Reply:
I think we can refer to the existing unit/integration tests for tf frontend. Most of them just create a tf graph.